projects/1.20-fabric/assets/cafes-birding/cafesbirding/lang/zh_cn.json

```
@@ -0,0 +1,146 @@
+{
+  "block.cafesbirding.iron_grate": "铁笼",
+  "block.cafesbirding.light_iron_grate": "抛光铁笼",
+  "painting.cafesbirding.sunflower.title": "向日葵",

```

```suggestion
  "painting.cafesbirding.sunflower.title": "Sunflower",
```
画名不应当翻译
此类问题请全文通查并修改

---

通查完毕，已修改